app/views/layouts/_signin_signout.html.erb

```diff
@@ -29,16 +29,16 @@
 
 <% else %>
   <% if !isActivePage(root_path) %>
-    <a href="#header-login-form" data-toggle="modal" dialog-width="450px"><%= _('Sign in')%></a> /
-    <a href="#header-register-form" data-toggle="modal" dialog-width="450px"><%= _('Create account')%></a>
+    <a id="sign_in_modal" href="#header-login-form" data-toggle="modal" dialog-width="450px"><%= _('Sign in')%></a> /
+    <a id="create_account_modal" href="#header-register-form" data-toggle="modal" dialog-width="450px"><%= _('Create account')%></a>
   <% end %>
 <% end %>
 
 <% if !user_signed_in? && !isActivePage(root_path) then %>
-  <div id="header-login-form" class="modal" style="display:none">
+  <div id="header-login-form" class="modal" role="dialog" aria-labelledby="#sign_in_modal" style="display:none">
```

Comment:
The doc says that the `aria-labelledby` attributes here are supposed to point to the `<h2>` on the partials.